cli/flags.rs

```diff
@@ -136,11 +137,25 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
     ).subcommand(
       SubCommand::with_name("xeval")
         .setting(AppSettings::DisableVersion)
-        .about("xargs-like eval script")
-        .arg(
-          Arg::with_name("var")
-            .long("var")
-            .short("V")
+        .about("`xargs`-like eval script")
```

Comment:
No backticks please

Also I don't think we should assume people know what xargs is. It would be clearer to write "eval a script on lines from stdin"